js/data/question.jsonc

```diff
@@ -0,0 +1,599 @@
+	"explanation": "詳細:",
+	"override_support_chance": "支援艦隊来援率上書き",
+	"normal_day": "道中支援(昼戦)",
+	"normal_night": "道中支援(夜戦)",
+	"boss": "決戦支援",
+	"restore_defaults": "設定をリセット",
+	"simulation": "Simulation", // ok
+	"watch_sortie": "出撃記録",
+	"opens_in_replayer": "プレーヤーで開く",
+	"simulate_n_times_and_show_stats": "シミュレート{inp_num}回の統計を表示:",
+	"go": "Go", // ok
+	"cancel": "Cancel", // ok
+	"clear": "Clear", // ok
+	"statistics": "演算結果",
+	"note_calc_wip": "計算の精度は現在も改善中で、新しい情報が発見されるにつれて改善されます。値は大まかなものであり、実際のゲーム内レートを正確に反映していない可能性があることに注意してください。",
+	"warn_combined_fleet": "警告: 連合艦隊と警戒陣の計算は非常に暫定的なため、これらの結果は特に大まかです。設定については、「詳細設定」を参照してください。",
+	"number_of_runs": "演算回数",
+	"final_rank_rate": "出撃あたりのの最終評価",
+	"retreat": "撤退",
+	"enemy_flagship_sunk_rate": "旗艦撃沈率",
+	"flagship_sunk": "旗艦撃沈",
+	"avg_n_hp_per_run": "出撃あたり{txt_hp}HP",
+	"avg_n_hp_per_boss": "ボス到達あたり{txt_hp}HP",
+	"avg_n_tp_per_run": "出撃あたり{txt_tp}TP",
+	"rank_per_battle": "戦闘ごとの戦闘評価",
+	"mvp_rate_per_battle": "{img_mvp}戦闘ごとのMVP率:",
+	"ship_n": "{0}隻目",
+	"air_state_rate_per_battle": "戦闘ごとの制空状態",
+	"air_state_ASP": "確保",
+	"air_state_AS": "優勢",
+	"air_state_AP": "拮抗",
+	"air_state_AD": "劣勢",
+	"air_state_AI": "喪失",
+	"taiha_rate_per_battle": "{img_taiha}戦闘ごとの大破率:",
+	"heavy_damage": "大破",
+	"any": "Any", // ok
+	"no_chuuha_after_battle": "戦闘後に{img_chuuha}艦がいない割合:",
+	"moderate_damage": "中破",
+	"all": "All", // ok
+	"avg_resupply": "平均補給コスト",
+	"fuel": "燃料",
+	"ammo": "弾薬",
+	"steel": "鋼材",
+	"baux": "ボーキ",
+	"bucket": "バケツ",
+	"repair_team": "ダメコン",
+	"avg_repair": "平均入渠コスト",
+	"avg_resource_per": "最終ノード{img}あたり:",
+	"copy_to_clipboard": "クリップボードにコピー",
+	"copied_to_clipboard": "コピーされました",
+	"avg_emptied_plane_slot": "平均艦載機スロット全滅数",
+	"avg_emptied_lbas_slot": "平均基地航空部隊全滅数",
+	
+	"node_n": "Node {0}",
+	"battle_type": "戦闘種別",
+    "battle_normal": "通常戦闘マス",
+    "battle_night_only": "開幕夜戦マス",
+    "battle_air_raid": "空襲戦マス",
+    "battle_air_only": "航空戦マス",
+	"night_battle": "夜戦する?",
+    "always_if_possible": "可能ならいつでも",
+    "only_day_rank_less_than_A": "A未満なら",
+    "only_day_rank_less_than_B": "B未満なら",
+    "only_flag_not_sunk": "敵旗艦が健在なら",
+    "resupply_cost": "資源消費タイプ",
+    "submarine_only": "潜水艦のみ",
+	"add_cost": "追加コスト(戦闘開始前に現在の残り資源から)",
+	"max": "Max", // ok
+	"player_formation": "自艦隊陣形",
+    "use_form_if_special_impossible": "特殊砲撃が不可能な場合、\u00a0{img_form}\u00a0を使用",
+	"line_ahead": "単縦陣",
+    "CF4": "第四警戒航行序列: 戦闘隊形",
+	"lbas_waves": "基地航空隊派遣",
+	"support": "支援艦隊",
+	"support_normal": "道中",
+	"support_boss": "決戦",
+	"use_smokescreen": "煙幕を使用",
+    "enable_barrage_balloon_effects": "阻塞気球を使用する",
+    "enable_atoll_node_effects": "特四式内火艇/改による越環礁攻撃を実施する",
+    "use_anchorage_repair_before_battle": "先頭前に緊急泊地修理を実施する",
+	"override_historical_bonuses": "史実ボーナス上書き",
+	"set_bonuses": "手動セット",
+	"main_fleet_set": "Main Fleet set", // ?
+	"lbas_set": "Land Base Aerial Support set", // ?
+	"friend_fleet_set": "Friend Fleet set", // ?

```

Comment:
"Main Fleet set" means "Main Fleet bonuses have been set", to notify the user that they have already set some value. Same thing for the other two.
"lbas_set"/"friend_fleet_set" are currently unused but maybe I may find a use in the future.